content/documentation/pages/4-batch-developer-guides/2-batch/8-standalone-simple-task-cloudfoundry.md

```diff
@@ -0,0 +1,289 @@
+---
+path: 'batch-developer-guides/batch/stanalone-simple-task-cloudfoundry/'
+title: 'Deploying a stand-alone task application on Cloud Foundry'
+description: 'Guide to deploying stand-alone spring-cloud-stream-task applications on Cloud Foundry'
+---
+
+# Deploying a task application to Cloud Foundry
+
+This guide will walk through how to deploy and run simple [spring-cloud-task](https://spring.io/projects/spring-cloud-task) stand-alone applications to Cloud Foundry.
+
+# Requirements
+
+On your local machine, you will need to have installed:
+
+- Java
+- [Git](https://git-scm.com/)
+
+Please also make sure that you have the [Cloud Foundry command line interface](https://console.run.pivotal.io/tools) installed ([documentation](https://docs.run.pivotal.io/cf-cli/)).
+
+## Project Setup
+
+The example consists of 2 applications:
+
+- billsetuptask
+- billrun
+
+The first task application `billsetuptask` will create an SQL table that is needed by the second task application.
+
+```sql
+CREATE TABLE IF NOT EXISTS BILL_STATEMENTS (
+    id int,
+    first_name varchar(50),
+    last_name varchar(50),
+    minutes int,
+    data_usage int,
+    bill_amount decimal(10,2)
+);
+```
+
+The second task (Spring Batch) application `billrun` will read JSON files containing simple user cellphone usage data from a specified directory, price it and place the results in the previously created database table.
+
+They are both located in the same Git repository. In order to get started, please checkout the project using:
+
+```bash
```

Comment:
Since we now have the copy ability, it was decided to remove the `$` prefix.   That way the use can copy and go.